app/src/main/java/com/sunasterisk/youtubebackground/data/source/remote/PageRemoteDataSource.kt

```diff
@@ -0,0 +1,26 @@
+package com.sunasterisk.youtubebackground.data.source.remote
+
+import com.sunasterisk.youtubebackground.data.model.Page
+import com.sunasterisk.youtubebackground.data.model.PageEntry
+import com.sunasterisk.youtubebackground.data.source.PageDataSource
+import com.sunasterisk.youtubebackground.data.source.remote.fetchjson.GetJsonFromUrl
+import com.sunasterisk.youtubebackground.utils.Constants
+
+class PageRemoteDataSource : PageDataSource.Remote {
```

Comment:
```suggestion
class VideoRemoteDataSource : PageDataSource.Remote {
```
Sao lại đặt tên là `PageRemoteDataSource`, cái này mình đang lấy dữ liệu Video mà nhỉ?

Reply:
cái này lấy 1 cái page trong đó có mấy thuộc tính với 1 list video nữa anh

Reply:
Quy lại thì nó vẫn là lấy danh sách Video nên nó là data souce về Video.
Mình đặt tên VideoDataSource có nghĩa là tất cả xử lý dữ liệu liên quan về Video chứ không phải về Page, page chỉ thông số phụ thôi.
Nên sửa tên lại các class luôn nghe